README.md

```diff
@@ -405,16 +405,35 @@ it will return `nil`.
 
 resty.kong.log.set\_log\_level
 ----------------------------------
-**syntax:** *resty.kong.log.set_log_level(level)*
+**syntax:** *resty.kong.log.set_log_level(level, timeout)*
 
-**context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, balancer_by_lua&#42;, ngx.timer.&#42;*
+**context:** *any*
```

Comment:
We still have `base.allows_subsystem("http")` in the lua source, so it should not be `any`.

Perhaps in the next version we will remove this limitiation.

Reply:
In the next line we have restricted the `subsystem`.

Reply:
Sorry, my mistake.